### PR TITLE
Update and rename spotify_qt-3.10.recipe to spotify_qt-3.12.recipe

### PR DESCRIPTION
**Pull request by Knezzen**
Version bump and added x86_64 architecture as tested.

### CODE REVIEW

File: media-sound/spotify_qt/spotify_qt-3.12.recipe [hunk 1]
```diff
@@ -8,12 +8,12 @@ COPYRIGHT="2020 Kraxarn"
...
 REVISION="1"
 SOURCE_URI="https://github.com/kraxarn/spotify-qt/archive/refs/tags/v$portVersion.tar.gz"
-CHECKSUM_SHA256="66191fc75f8571f40be16582caded2e6e3181434cbab5e85d3e9613c97ecb44f"
+CHECKSUM_SHA256="5782ff9cd854aa92b4ba49837d3ed4b5ada0427fd440e16866e9727abcd9df62"
 SOURCE_FILENAME="spotify-qt-v$portVersion.tar.gz"
 SOURCE_DIR="spotify-qt-$portVersion"
 ADDITIONAL_FILES="spotify_qt.rdef.in"
 
-ARCHITECTURES="?all !x86_gcc2"
+ARCHITECTURES="?all x86_64 !x86_gcc2"
```
Comment (Begasus) [inline media-sound/spotify_qt/spotify_qt-3.12.recipe:16]: As there are only 2 buildmasters currently, we don't need to disable other architectures here. :)

```suggestion
ARCHITECTURES="all !x86_gcc2"
```

**Comment by Begasus** [comment]
Thanks for looking into this, haven't touched it in a long time, did you actually got it up and running as it should?
PS, checked both architectures for the build?

**Comment by Knezzen** [comment]
I have only tested x86_64, hence the "?all" still being there. I got it up and running as it should though on x86_64.

**Comment by Begasus** [comment]
OK, didn't spot this was still disabled for 32bit, lets merge, thanks!

**Pull request merged**